python_src/src/lamp_py/performance_manager/pipeline.py

```diff
@@ -66,18 +85,23 @@ def iteration() -> None:
         process_logger.log_start()
 
         try:
-            process_static_tables(db_manager)
-            process_gtfs_rt_files(db_manager)
-            write_flat_files(db_manager)
-            start_parquet_updates(db_manager)
+            # temporary check to see if legacy metadata exists. if it does,
+            # raise an exception and try again on the next cycle.
+            if legacy_metadata_exists(rpm_db_manager):
+                raise EnvironmentError("Legacy Metadata Detected")
```

Comment:
raise an exception if there is legacy metadata. this will prevent us from running any of the performance manager logic, waiting for the next loop to give it a shot.